pkg/builders/s2i/builder_test.go

```diff
@@ -150,6 +150,59 @@ func Test_BuilderImageConfigurable(t *testing.T) {
 	}
 }
 
+// Test_BuildImageWithFuncIgnore ensures that ignored files are not added to the func
+// image
+func Test_BuildImageWithFuncIgnore(t *testing.T) {
+
+	funcIgnoreContent := []byte(`#testing comments
+hello.txt
+`)
+	f := fn.Function{
+		Runtime: "node",
+	}
+	tempdir := t.TempDir()
+	f.Root = tempdir
+	//create a .funcignore file containing the details of the files to be ignored
+	err := os.WriteFile(filepath.Join(f.Root, ".funcignore"), funcIgnoreContent, 0644)
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
It might be also interesting to create file named `#testing comments` and see if it is ignored (it should not).